Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java

```diff
@@ -281,6 +279,12 @@ public class CommonJsInterface extends JBridge implements in.juspay.hypersdk.cor
     private double lastLongitudeValue;
     private int lastFocusedEditView;
     private Polyline polylines = null; //TODO Implement polylies using key-value pair
+    private Polyline overlayPolylines = null;
+
+    private AnimatorSet polylineAnimatorSet = null;
+    private ValueAnimator polylineColorFadingAnimator = null;
+    private  ValueAnimator polylineDrawingAnimator = null;
+    private boolean isAnimationNeeded = false;
```

Comment:
do we need all of  these ?

Reply:
we can directly use these from function instead of defining globally

Reply:
we are using these functions in different functions. they are independent to each other. So it should be global.

Reply:
okay